doc/source/conf.py

```diff
@@ -228,6 +228,7 @@
     (r'py:.*', 'BackfaceArgs'),
     (r'py:.*', 'CullingOptions'),
     (r'py:.*', 'OpacityOptions'),
+    (r'py:.*', 'CameraPosition'),
```

Comment:
I think this needs to include `Options`
```suggestion
    (r'py:.*', 'CameraPositionOptions'),
```